index.js

```diff
@@ -228,14 +245,20 @@ Parser.parseURL = function(feedUrl, settings, callback) {
       xml += chunk;
     });
     res.on('end', function() {
-      return Parser.parseString(xml, callback);
+      return Parser.parseString(xml, settings, callback);
     })
   })
   req.on('error', callback);
 }
 
-Parser.parseFile = function(file, callback) {
+Parser.parseFile = function(file,settings,callback) {
+
+  if (!callback) {
```

Comment:
No need for this block here, since settings isn't used until we get to `parseString`.